spec/clean_localization/client_spec.rb

```diff
@@ -4,6 +4,32 @@
   let(:lang) { 'en' }
   let(:client) { described_class.new(lang) }
 
+  describe '.data' do
+    it 'loads data only once, caching it' do
+      expect(CleanLocalization::Config)
+        .to receive(:load_data)
+        .and_return({})
+        .once
+
+      expect(described_class.data).to eq({})
+      expect(described_class.data).to eq({})
+    end
+  end
+
+  describe '.reload_data!' do
+    it 'reloads data into `data`' do
+      pre_data = CleanLocalization::Client.data # ensure data is loaded
+
+      expect do
+        CleanLocalization::Config.base_path = Dir.mktmpdir
+        described_class.reload_data!
+      end
```

Comment:
a syntax like `end.to` is not readable. Can you extract 2 lines into lambda and convert this expect to `expect(&my_new_lambda)` or `expect { my_new_lambda.call }`

Reply:
Good point!